### PR TITLE
Make bottom nav

### PR DESCRIPTION
**Pull request by thefrankharvey**
Please sir, may I merge now

### CODE REVIEW

**Review by MatthewDtotheG — COMMENTED** [review]
Does the useContext hook not get the job done here? const {theme} = useContext(ThemeContext);

**Comment by thefrankharvey** [comment]
> Does the useContext hook not get the job done here? const {theme} = useContext(ThemeContext). You should be able to do theme.Home or theme.ThemeTwo without needing the useEffect or switchTheme function

I can try it

File: src/components/Carousel.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+export const Carousel = (props) => {
+
+  const { items, style } = props;
+  const itemsPerInterval = props.itemsPerInterval === undefined
+    ? 1
+    : props.itemsPerInterval;
+
+  const [interval, setInterval] = React.useState(1);
+  const [intervals, setIntervals] = React.useState(1);
+  const [width, setWidth] = React.useState(0);
```
Comment (MatthewDtotheG) [inline src/components/Carousel.js]: This is a small thing but whenever we use hooks I would like to import them like this

`import React, { useState, useEffect } from 'react';`

Comment (MatthewDtotheG) [reply src/components/Carousel.js]: So `const [interval, setInterval] = useState(1);`

**Pull request merged**